Heroes.ReplayParser/Player.cs

```diff
@@ -48,7 +48,7 @@ public class Player
         /// <summary>
         /// Gets or sets the BattleTag (Numbers only)
         /// </summary>
-        public int BattleTag { get; set; }
+        public int? BattleTag { get; set; } = null;
```

Comment:
I'd rather not change this to nullable.  I think it's fine to just be '0' in this extremely rare scenario.
```suggestion
        public int BattleTag { get; set; }
```